fem/bilininteg_hcurl.cpp

```diff
@@ -4843,6 +4844,1923 @@ void MixedVectorWeakCurlIntegrator::AddMultPA(const Vector &x, Vector &y) const
    }
 }
 
+// Apply to x corresponding to DOFs in H^1 (domain) the (topological) gradient
+// to get a dof in H(curl) (range). You can think of the range as the "test" space
+// and the domain as the "trial" space, but there's no integration.
+static void PAHcurlApplyGradient2D(const int c_dofs1D,
+                                   const int o_dofs1D,
+                                   const int NE,
+                                   const Array<double> &_B,
```

Comment:
We should remove all the `_*` variable names to avoid errors with preprocessors and `cygwin`.

Reply:
@camierjs I will fix this, but I don't understand why this is an issue if there are so many occurrences beyond this PR. If you do `grep "&\_" *` in `mfem/fem` you will find many occurrences in many files, in the master branch. Does the master branch not give errors? Should I fix all of them or just the ones in this PR?

Reply:
I was just looking at #1828 and thinking that we should not add some more. I agree that we will have to go all over the code to sanitize this names. Just the ones in this PR would be great.

Reply:
Thanks for clarifying; now it makes sense. I think I fixed all instances introduced in this PR, with the latest commit.

Reply:
To be clear, the issue is with the *leading* underscore `_something`. Trailing underscores `something_` are fine.

Reply:
@barker29 Thanks for underscoring that.